module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -473,33 +517,49 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::goalieWalk(const Field& field, const Ball& ball) {
+
+        // make a motionCommand to emit
         std::unique_ptr<MotionCommand> motionCommand;
 
+        // time since the ball has been seen, in microseconds, current clock - time the ball was last measured
         float timeSinceBallSeen =
             std::chrono::duration_cast<std::chrono::microseconds>(NUClear::clock::now() - ballLastMeasured).count()
-            * 1e-6;
+            * 1e6;  // SUSS
```

Comment:
I wrote this comment hahahaha how did it make it in here. 

It is super suss. The duration cast already converts it to microseconds. This would (if I'm not mistaken) convert it to picoseconds.